src/librustc_mir/diagnostics.rs

```diff
@@ -335,6 +335,8 @@ match Some(42) {
 "##,
 
 E0162: r##"
+#### Note: this error code is no longer emitted by the compiler.
+
 An if-let pattern attempts to match the pattern, and enters the body if the
 match was successful. If the match is irrefutable (when it cannot fail to
 match), use a regular `let`-binding instead. For instance:
```

Comment:
You need to modify lines 344 and 373. As the error is no longer emitted, the test suite catches that E0162 and E0165 didn't fire. They should probably now be `compile_pass` or similar.